src/UsdPlusRedeemer.sol

```diff
@@ -224,21 +224,6 @@ contract UsdPlusRedeemer is IUsdPlusRedeemer, UUPSUpgradeable, AccessControlDefa
         emit RequestCancelled(ticket, request.receiver);
 
         // return USD+ to requester
-        IERC20($._usdplus).safeTransfer(request.owner, request.usdplusAmount);
-    }
-
-    /// @notice Fulfills request to burn USD+ without sending payment token
-    /// @dev This is a special case for USD+ bridging and 0 payment redemption
-    function burnRequest(uint256 ticket) external onlyRole(FULFILLER_ROLE) {
-        UsdPlusRedeemerStorage storage $ = _getUsdPlusRedeemerStorage();
-        Request memory request = $._requests[ticket];
-
-        if (request.receiver == address(0)) revert InvalidTicket();
-
-        delete $._requests[ticket];
-
-        emit RequestBurned(ticket, request.receiver, request.usdplusAmount);
-
-        UsdPlus($._usdplus).burn(request.usdplusAmount);
+        UsdPlus($._usdplus).mint(request.owner, request.usdplusAmount);
```

Comment:
Does this check for duplicate cancel calls on the request? Don't want to allow someone to mint twice

Reply:
Yes, if cancel was called previously for this request, it will error with `InvalidTicket`